scripts/merge-config.escript

```diff
@@ -12,16 +12,29 @@
 
 main(_) ->
     {ok, BaseConf} = file:read_file("apps/emqx_conf/etc/emqx_conf.conf"),
+
     Cfgs = get_all_cfgs("apps/"),
-    Conf = lists:foldl(fun(CfgFile, Acc) ->
-                               case filelib:is_regular(CfgFile) of
-                                   true ->
-                                       {ok, Bin1} = file:read_file(CfgFile),
-                                       [Acc, io_lib:nl(), Bin1];
-                                   false -> Acc
-                               end
-                       end, BaseConf, Cfgs),
-    ok = file:write_file("apps/emqx_conf/etc/emqx.conf.all", Conf).
+    Conf = [merge(BaseConf, Cfgs),
+            io_lib:nl(),
+            "{{ disable_entrerprise_conf }}include emqx_enterprise.conf",
```

Comment:
there is no need for this.
just `include "emqx_enterprise.conf"` should work because HOCON silently ignores missing-file includes.

Reply:
https://github.com/zmstone/emqx/commit/36f78ea2ee9d510ef51d38c683c8a8ec4c2b6415